docs/guides/user/sigma.md

```diff
@@ -64,7 +78,13 @@ timesketch/data/sigma
 
 ### Sigma Rules Blocklist file
 
-The `data/sigma_blocklist.csv` is where Timesketch maintains a list of incompatible rules. By default each rule is considered compatible, but it is good practice to add them in this file if they are tested and verified to not be compatible.
+The `data/sigma_blocklist.csv` is where Timesketch maintains a list of rules. 
+Each rule can have one of the following status: `good,bad,exploratory`.
```

Comment:
How about adding `values` after the `status` word so it becomes `the following status values:` ?